File: Solutions/3409/java/program.java
// Copy from Quera

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        for(int i = 1; i <= n; i++)
        {
            for (int j = 1; j <= n; j++)
                System.out.print(i * j + " ");
            System.out.println(" ");
        }
    }
}

File: Solutions/3409/c++/program.cpp
// Copy from Quera

#include <iostream>
using namespace std;

int main()
{
    int n;
    cin >> n;
    for (int i = 1; i <= n; i++)
    {
        for (int j = 1; j <= n; j++)
            cout << i * j << ' ';
        cout << '\n';
    }
    return 0;
}
